.golangci.yml

```diff
@@ -114,3 +118,7 @@ issues:
     - path: services\.go
       linters:
         - dupl
+    # Run wrapcheck only on resources files.
```

Comment:
A better configuration would be to either have `include-rules` instead of excludes, or use negative lookahead pattern. None seems to be supported by the linter